dogs.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ADOPTADOG</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+  </head>
+
+  <body>
+      <header>
+        <div class='float-right'>
```

Comment:
try to not use float for laying out this kind of thing. We want to limit the uses of floats as much as we can. Also, in your HTML use double quotes for your attributes

```
<div class="float-right">
```